README.md

```diff
@@ -1,20 +1,30 @@
+# MongoDB Helm Charts repository for Kubernetes
 
-## MongoDB Helm Charts repository for Kubernetes
+## Trial Version of Helm Charts
 
-## Charts
+This repository contains Helm Charts for different MongoDB products. Currently,
+only the MongoDB Atlas Operator is supported (in _Trial mode_).
 
-This repository contains sample HELM charts for different MongoDB products
+| Charts                                                         | Description                                                                                                                      |
+| -------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| [atlas-operator](charts/atlas-operator)                        | MongoDB Atlas Operator Helm Chart. _Start Here!_                                                                                 |
```

Comment:
The table looks ugly on small screens without preview. Since characters might have different lengths (and will misalign with some fonts) I suggest to use pipes as separators and do not try to build an actual table.

P.S. we can also change "Start Here!" into a link to make it actionable.
```
| Charts                 | Description                                         |
| ---------------------- | --------------------------------------------------- |
| [atlas-operator](charts/atlas-operator) | MongoDB Atlas Operator Helm Chart. [_Start Here!_](charts/atlas-operator) |
| [atlas-cluster](charts/atlas-cluster) | MongoDB Atlas Cluster Helm Chart. Create Mongo Database resources. |
| (_Optional_) [atlas-operator-crds](charts/atlas-operator-crds) | MongoDB Atlas Custom Resource Definitions (CRDs) Helm Chart. Installed automatically by [atlas-operator](charts/atlas-operator). |
```